DNN.Survey/App_LocalResources/SurveyResults.ascx.resx

```diff
@@ -117,9 +117,15 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="ControlTitle_surveyresults.text" xml:space="preserve">
+    <value>Ergebnisse</value>
```

Comment:
This would should be english

Reply:
I changed it and commited it directly to the pull request.